Doc/library/uuid.rst

```diff
@@ -11,9 +11,10 @@
 --------------
 
 This module provides immutable :class:`UUID` objects (the :class:`UUID` class)
-and the functions :func:`uuid1`, :func:`uuid3`, :func:`uuid4`, :func:`uuid5`,
-:func:`uuid6`, and :func:`uuid8` for generating version 1, 3, 4, 5, 6,
-and 8 UUIDs as specified in :rfc:`9562` (which supersedes :rfc:`4122`).
+and :ref:`functions <uuid-factory-functions>` for generating UUIDs corresponding
+to a specific UUID version as specified in :rfc:`9562` (which supersedes :rfc:`4122`),
+e.g., :func:`uuid1` for UUID version 1, :func:`uuid3` for UUID version 3, and so on.
```

Comment:
https://devguide.python.org/documentation/style-guide/#use-simple-language

```suggestion
for example, :func:`uuid1` for UUID version 1, :func:`uuid3` for UUID version 3, and so on.
```

Reply:
I'm sorry, I'm too used to that formulation :')